src/pineko/evolve.py

```diff
@@ -172,21 +190,37 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
             "are you sure that's what you want?"
         )
 
-    with open(card_path, "w", encoding="UTF-8") as f:
-        yaml.safe_dump(operators_card, f)
-        f.write(f"# {pineko_version=}")
+    def dump_card(card_path, operators_card, conv_type):
+        operators_card["configs"]["polarized"] = conv_type == "polPDF"
+        with open(card_path, "w", encoding="UTF-8") as f:
+            yaml.safe_dump(operators_card, f)
+            f.write(f"# {pineko_version=}")
+
+    # For hardonic obs we might need to dump 2 eko cards
+
+    if conv_type_a == conv_type_b:
+        dump_card(card_path, operators_card, conv_type_a)
+    else:
+        # dump eko1
+        card_name = f"{card_path.stem}_{conv_type_a}.yaml"
+        dump_card(card_path.parent / card_name, operators_card)
```

Comment:
Move the construction of `card_name` into the `dump_card` function. Same for below.

This should be enough to fix the tests and benchmarks.
```suggestion
        dump_card(card_path.parent / card_name, operators_card, conv_type_a)
```

Reply:
But we agreed not to change the name of the card for the dafault case with a single eko... 
To make bench passing we need a new pineappl version.